Dockerfile.pi

```diff
@@ -0,0 +1,21 @@
+FROM node:14-alpine as build-stage
+
+ENV NODE_OPTIONS=--max-old-space-size=512
```

Comment:
For the pi Dockerfile, this is needed as sometimes you can get a heap out of memory issue on the npm run build step